test/component.spec.ts

```diff
@@ -84,11 +95,9 @@ test("subraph component", function ({ components, stubComponents }) {
 
         it("should throw the appropiate error", async () => {
           const { subgraph } = components
-          try {
-            await subgraph.query("query", {}, 0) // no retires
-          } catch (error) {
-            expect(error.message).toBe(`Invalid request. Status: ${response.status}`)
-          }
+          await expect(() => subgraph.query("query", {}, 0)).rejects.toThrow(
+            `Invalid request. Status: ${response.status}`
+          )
```

Comment:
If you remove the `() =>` the test should keep working, as the expectation will be performed over a promise that will be processed by the reject clause.
```suggestion
          await expect(subgraph.query("query", {}, 0)).rejects.toThrow(
            `Invalid request. Status: ${response.status}`
          )
```

Reply:
good catch, will change